examples/preprocessing/plot_all_scaling.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
+=============================================================
+Compare the effect of different scalers on data with outliers
+=============================================================
+
+The feature 0 and feature 5 of california housing dataset contains large
+outliers that can make visualization of the data difficult.
+
+Also linear models like :class:`sklearn.linear_model.SVM` require data which is
+normalized to the range [-1, 1].
+
+This example uses different scalers and normalizers to bring the data within a
+smaller range.
+
+# TODO make doc better
+"""
+from __future__ import print_function
+print(__doc__)
+
+# Author: Thomas Unterthiner
+#         Raghav RV <rvraghav93@gmail.com>
+# License: BSD 3 clause
+
+import numpy as np
+
+import matplotlib as mpl
+from matplotlib import pyplot as plt
+from matplotlib import cm, gridspec
+
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.preprocessing import minmax_scale
+from sklearn.preprocessing import MaxAbsScaler
+from sklearn.preprocessing import StandardScaler
+from sklearn.preprocessing import RobustScaler
+from sklearn.preprocessing import Normalizer
+from sklearn.preprocessing.data import QuantileNormalizer
+
+from sklearn.datasets import fetch_california_housing
+
+dataset = fetch_california_housing()
+X_full, y_full = dataset.data, dataset.target
+
+# Take only 2 features to make visualization easier
+# Feature 0 has a tapering distribution of outliers
```

Comment:
Feature of 0 has a large tail distribution. Feature 5 has a few but very large outliers.